apps/docs/components/ui/sectionPopover/sectionPopover.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+
+import { useState } from "react";
+import cx from "classnames";
+import { Button } from "react-aria-components";
+
+import "./sectionPopover.css";
+import ChevronIcon from "./assets/chevron-icon.svg";
+
+interface Link {
+    title: string;
+    url: string;
+    id: string;
+}
+
+interface SectionPopoverProps {
+    links: Link[];
+}
+
+const SectionPopover = ({ links }: React.PropsWithoutRef<SectionPopoverProps>) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const togglePopover = () => {
+        setIsOpen(!isOpen);
+    };
+
+    const handleWindowResize = () => {
+        if (window.innerWidth > 768) {
+            setIsOpen(false);
+        }
+    };
+
+    window.addEventListener("resize", handleWindowResize);
+
+    const listItems = links.map(link => (
+        <li className="hd-section-popover__list-item" key={link.id}>
+            <a href={link.url} className="hd-section-popover__list-link" onClick={togglePopover}>
+                {link.title}
+            </a>
+        </li>
+    ));
+
+    return (
```

Comment:
oh I think you're error might actually be from listItems.length > 0 && (
You should add <></> around this. if that's false, it returns nothing.